src/data/data.ts

```diff
@@ -322,6 +322,14 @@ export const RiskProfile: { [key: string]: PersonRiskValue } = {
   },
 }
 
+// This is an explicit list of risk profiles for which "their vaccine" modifiers
+// apply.
+export const VaccineModifiableRiskProfiles: (keyof typeof RiskProfile)[] = [
+  'average',
+  'livingAlone',
```

Comment:
Ostensibly this should work for everything except the fixed value profiles?

Reply:
that's everything except the fixed microcovid budgets and "literally has covid"?

Reply:
Yeah, I think so. Thinking aloud here:

 - calculated profiles (not average or fixed) are determined by doing activities with random people and summing the microCOVIDS
 - risk from doing activities = person risk * activity risk
 - vaccine changes activity risk

So I don't see why the vaccine math that applies to livingAlone wouldn't apply to goes to bars?